tutorials/haskell/hspec-golden-test-tutorial/code/ChangeLog.md

```diff
@@ -0,0 +1,3 @@
+# Changelog for hspec-golden-tests
+
+## Unreleased changes
```

Comment:
@Jagl257 I think we can remove this file.

Reply:
Yes Cris, alredy done!